steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019MCRecon_KF_WithSpacing.lcsim

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcsim xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcsim/1.0/lcsim.xsd">
+    <!-- 
+      @brief Steering file that will be used for the reconstruction of 2019 MC. 
+      @author <a href="mailto:omoreno1@ucsc.edu">Omar Moreno</a>
+    -->
+    <execute>
+        <driver name="EventMarkerDriver"/>
+        <!-- Ecal reconstruction drivers -->        
+        <driver name="EcalRawConverter" />
+        <driver name="EcalTimeCorrection"/>
+        <driver name="ReconClusterer" />
+        <driver name="CopyCluster" />
+        <driver name="HodoRunningPedestal"/>
+        <driver name="HodoRawConverter"/>
+        <!-- SVT hit reconstruction drivers -->
+        <driver name="RawTrackerHitSensorSetup"/>
+        <driver name="RawTrackerHitFitterDriver" />
+        <driver name="TrackerHitDriver"/>
+
+        <driver name="KalmanPatRecDriver"/> 
+        <driver name="TrackTruthMatching_KF" />
+        <driver name="ReconParticleDriver_Kalman" />
+	<driver name="KFOutputDriver"/>
+
+        <driver name="LCIOWriter"/>
+        <driver name="CleanupDriver"/>
+    </execute>    
+    <drivers>
+        <driver name="EventMarkerDriver" type="org.lcsim.job.EventMarkerDriver">
+            <eventInterval>1000</eventInterval>
+        </driver>
+        <!-- Ecal reconstruction drivers -->
+        <driver name="EcalRawConverter" type="org.hps.recon.ecal.EcalRawConverter2Driver"/>
+        <driver name="EcalTimeCorrection" type="org.hps.recon.ecal.EcalTimeCorrectionDriver"/>
+        <driver name="ReconClusterer" type="org.hps.recon.ecal.cluster.ReconClusterDriver">
+            <logLevel>WARNING</logLevel>
+            <outputClusterCollectionName>EcalClusters</outputClusterCollectionName>
+            <hitEnergyThreshold>0.030</hitEnergyThreshold>
```

Comment:
Are these MC specific? We don't configure an energy threshold in our data config.

Reply:
Removed